src/database.js

```diff
@@ -83,4 +84,14 @@ export class Database {
 
     return id;
   }
+
+  _update(collectionName, record) {
+    const originalRecord = this.find(collectionName, {id: record.id});
```

Comment:
Nope, const in ES6 means that reference to object that this variable is pointing to can't change, not properties of this object itself :)